cli/cmdlineparser.cpp

```diff
@@ -148,9 +148,10 @@ bool CmdLineParser::fillSettingsFromArgs(int argc, const char* const argv[])
     if (success) {
         if (getShowErrorMessages()) {
             XMLErrorMessagesLogger xmlLogger;
-            std::cout << ErrorMessage::getXMLHeader(mSettings.cppcheckCfgProductName);
+            ErrorLogger& log = xmlLogger;
```

Comment:
I chose this so we keep using the interface. The actual implementation is private and I would like to keep it that way.